docs/notebook_tutorial/ocr.ipynb

```diff
@@ -271,10 +291,10 @@
       "text/plain": [
-       "<PIL.JpegImagePlugin.JpegImageFile image mode=RGB size=810x539 at 0x7FE82F0AA410>"
+       "<PIL.JpegImagePlugin.JpegImageFile image mode=RGB size=810x539 at 0x7F3A107FBA90>"
```

Comment:
In general, we want to keep the repo (master branch) lean and don't push a lot of binary files to it. These encoded images are also taken up space.

However, I noticed most images are already committed in a prior PR. This means the size of the repo won't reduce if we delete them. Let's just keep them there. 

Just keep in mind that next time let's find a better way to store the actual images, maybe in the `asset` branch

Reply:
I already stored the source picture in `asset`. The pictures shown are code cell outputs.

Reply:
yes, the code cell output actually also takes space. We can keep this in mind next time. Generally speaking, binary values are not suitable for git.